zuul.d/jobs.yaml

```diff
@@ -1787,6 +1787,69 @@
     vars:
       ansible_collections_repo: github.com/ansible-collections/frr.frr
 
+- job:
+    name: ansible-test-sanity-yang
+    parent: ansible-test-sanity-base
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    timeout: 3600
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python27
+    parent: ansible-test-units-base-python27
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python35
+    parent: ansible-test-units-base-python35
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python36
+    parent: ansible-test-units-base-python36
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python37
+    parent: ansible-test-units-base-python37
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python38
+    parent: ansible-test-units-base-python38
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
```

Comment:
lets drop integration jobs to start, this way we can properly test them before merging